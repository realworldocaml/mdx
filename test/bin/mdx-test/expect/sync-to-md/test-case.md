Mdx can also understand ocaml code blocks:


```ocaml file=sync_to_md.ml,part=toto
# let x = 3;;
val x : int = 3
# let y = 4;;
val y : int = 4
# print_int x;;
3
- : unit = ()
# print_int y;;
4
- : unit = ()
```

```ocaml file=sync_to_md.ml,part=zzz
```

```ocaml file=sync_to_md.ml,part=42
```

```ocaml file=sync_to_md.ml,part=
```

```ocaml file=sync_to_md.ml
```

```ocaml
# let x = 2;;
val x : int = 2
# print_int x;;
3
- : unit = ()
```
