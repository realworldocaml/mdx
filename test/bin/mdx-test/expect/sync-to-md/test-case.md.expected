Mdx can also understand ocaml code blocks:


```ocaml file=sync_to_md.ml,part=toto
let x = 34
let f = 42.3
let s = "toto"
let f x u = u x

let () =
  print_int x;
  print_float f
;;
```

```ocaml file=sync_to_md.ml,part=zzz
let () =
  print_string s
;;
```

```ocaml file=sync_to_md.ml,part=42
let () =
  f x print_int
```

```ocaml file=sync_to_md.ml,part=
let () =
  ();
  ()
;;
```

```ocaml file=sync_to_md.ml
let () =
  ();
  ()
;;

let x = 34
let f = 42.3
let s = "toto"
let f x u = u x

let () =
  print_int x;
  print_float f
;;

let () =
  print_string s
;;

let () =
  f x print_int
```

```ocaml
# let x = 2;;
val x : int = 2
# print_int x;;
2
- : unit = ()
```
